application/templates/cms/edit_measure_page.html

```diff
@@ -305,16 +305,6 @@ <h3 class="heading-medium">Data</h3>
 
             </div>
         </div>
-
-        <div class="grid-row">
-            <div class="column-two-thirds">
-                <h3 class="heading-medium">Contact details</h3>
-                <p>Contact details are for internal use only, and will not be displayed on the website.</p>
-                  {{ render_contact_details_fields(form.contact_name, form.contact_email, form.contact_phone, legend="Contact", further_explanation=False, disabled=form_disabled) }}
```

Comment:
Think we also need to get rid of the import of `render_contact_details_fields` at the top of this file, or Jinja should complain when it tries to render it.

Reply:
Although apparently it doesn't complain, because the tests pass...